upload.go

```diff
@@ -214,9 +216,35 @@ func (u Upload) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhttp
 	}
 	defer file.Close()
 
+	concatDir := u.DestDir
+
+	if u.CreateUuidDir {
+		uuidDir := uuid.New()
+
+		// It's very unlikely that the uuidDir already exists, but just in case
+		for {
+			if _, err := os.Stat(concatDir + "/" + uuidDir.String()); os.IsNotExist(err) {
+				break
+			} else {
+				uuidDir = uuid.New()
+			}
+		}
+
+		concatDir = concatDir + "/" + uuidDir.String()
+
+		if err := os.MkdirAll(concatDir, 0755); err != nil {
```

Comment:
Please can you move this block outside of `if u.CreateUuidDir {` so that the directories are also created when the `CreateUuidDir` isn't true.

Reply:
Fixed :)